lpr.Data/PackageData.cs

```diff
@@ -24,14 +23,36 @@ public Task<List<Package>> GetTopPackagesAsync(int amount)
       throw new NotImplementedException();
     }
 
-    public async Task<Package> CreatePackageAsync(Package newPackage) {
-      // Do we also need to create the first 'fallback' version?
+    public async Task<List<Package>> GetPackagesFromOrganisationAsync(Guid organisationId) {
+      Organisation organisation;
+      try {
```

Comment:
Maybe batter to not include a Try/catch here. You can validate "organization" on "!= null" and when null throw an exception

Reply:
Changed it 👍